spot-controller/App.js

```diff
@@ -127,7 +132,20 @@ export default class App extends React.Component {
      */
     _renderSettingsMenu() {
         return (
-            <SettingsMenu onClearRemoteUrl = { this._onClearRemoteUrl } />
+            <SettingsMenu
+                onClearRemoteUrl = { this._onClearRemoteUrl }
+                onResetApp = { this._onResetApp } />
         );
     }
+
+    /**
+     * Ah shit.
```

Comment:
:)

Reply:
lol

Reply:
This is what my code looks like without its makeup.